### ISSUE TITLE
bug 'freq_line' missing in intra burst 

### ISSUE THREAD

**Issue by agrouaze** [umr-lops/xsar_slc#93]
issue on 3 subswath in run v1.4c:
```python
Traceback (most recent call last):
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/bin/L1B_xspectra_IW_SLC_IFR", line 8, in <module>
    sys.exit(main())
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 169, in main
    generate_IW_L1Bxspec_product(slc_iw_path=slc_iw_path, output_filename=output_filename,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 92, in generate_IW_L1Bxspec_product
    one_subswath_xspectrum_dt = proc.compute_subswath_xspectra(dt, polarization=polarization.upper(),
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/xspectra.py", line 38, in compute_subswath_xspectra
    intra_xs = compute_IW_subswath_intraburst_xspectra(dt, polarization=polarization, tile_width=tile_width_intra,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/xspectra.py", line 169, in compute_IW_subswath_intraburst_xspectra
    burst_xspectra = tile_burst_to_xspectra(burst, dt['geolocation_annotation'], dt['orbit'], dt['calibration'],
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/intraburst.py", line 181, in tile_burst_to_xspectra
    xspecs = compute_intraburst_xspectrum(mod, mean_incidence, slant_spacing, azimuth_spacing,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/intraburst.py", line 307, in compute_intraburst_xspectrum
    np.fft.fftfreq(out.sizes['freq_' + azimuth_dim], azimuth_spacing / (out.attrs.pop('look_width') * 2 * np.pi))),
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/utils.py", line 458, in __getitem__
    return self.mapping[key]
KeyError: 'freq_line'
INFO:prun:[2023-03-23T17:55:18Z] End of processing for input line 1 : '/home/datawork-cersat-public/project/mpc-sentinel1/data/esa/sentinel-1a/L1/IW/S1A_IW_SLC__1S/2023/009/S1A_IW_SLC__1SDV_20230109T170108_20230109T170138_046709_059964_09F8.SAFE/measurement/s1a-iw1-slc-vh-20230109t170108-20230109t170136-046709-059964-001.tiff tiles18km 1.4c'. EXIT_CODE=1
```

**Comment by lanougue** [comment]
Bug confirmed. I found origin of the bug. work in progress

**Comment by lanougue** [comment]
solved in #97
